scheduler/db/scheduler.changelog-master.yaml

```diff
@@ -228,4 +233,160 @@ databaseChangeLog:
             columnName: guid
             defaultNullValue: "1"
             tableName: app_scaling_specific_date_schedule
+   - changeSet:
+      id: 7
+      author: aqan213
+      dbms: mysql
+      changes:
+        - createTable:
+            tableName: app_scaling_specific_date_schedule
+            columns:
+              - column:
+                  name: schedule_id
+                  type: bigint
```

Comment:
Myabe my knowledge on db site is not enough, but why change `bigint` to `bigserial` in postgres, and use `bigint` again in mysql ?

Reply:
Now all the `bigint` will not be changed in postgres instead of setting the default value of `schedule_id` to `nextval('schedule_id_sequence')` .

Reply:
we may still need to create `app_scaling_active_schedule`, see changeset 5.   Also, I wonder why missing `app_scaling_active_schedule` won't result in UT test failure.  Can you check the related UT as well?

Reply:
@cdlliuy There is no dbms condition for app_scaling_active_schedule, so it will be created for all dbms.